script/Ciel/Day 14/14_01A_CIEL14A_2.txt

```diff
@@ -445,22 +487,24 @@ What is this? What is this? What is this―――?!"
 [sha:b6e7fa4f9fff70dea6c9f1965079aa063c99510e]{
 -- Page 43, Offset 37737.
 -- 先ほどまでの血の海の中でのひとり遊びとは違う。
-This is different from when she was playing around in the pool of blood earlier.
+This is different to the way she was writhing around in the pool of blood earlier.
 }
 [sha:8b66285855284b364575f90638c08feae74c7fb9]{
 -- Page 44, Offset 37738.
 -- アルクェイドは自分の体を貫通した杭を引き抜こうともがくが、血に濡れた手は滑るばかりで杭を<僅|わず>かにしか動かせない。
-Arcueid struggles and tries to extract the stake that pierced her body, but the blood drenching her hands make her slip, allowing her to only move the stake slightly.
+Arcueid struggles to extract the stake that has pierced her,
+ but her hands are slippery with blood,
+ and she's barely able to move it at all.
```

Comment:
but her bloodstained hands keep slipping, and she's barely able to budge it at all.